### ISSUE TITLE
clean-up nfs exports on destroy

### ISSUE THREAD

**Issue by akostadinov** [mitchellh/vagrant#5701]
On vagrant destroy entries in /etc/exports are not removed. I believe it makes sense to have entries removed on `destroy`. I see #2738 but it seems it only cleans up invalid entries on vagrant up. I don't see why random other VMs run on same IP should have access to the shared dir of previously destroyed VM.

**Comment by christopher-hopper** [comment]
I am having this is the same problem. The error reported is:

    ==> local: Preparing to edit /etc/exports. Administrator privileges will be required...
    cp: missing destination file operand after ‘/etc/exports’
    Try 'cp --help' for more information.
    cp: cannot stat ‘/exports’: No such file or directory

Then, when I `vagrant halt` the machine, the nfs entries added by vagrant to the host `/etc/exports` file are not removed, leading to duplicates on a subsequent call to `vagrant up` or `vagrant reload` calls.

**host os:** Linux Ubuntu 14.04 LTS  
**vagrant:** 1.7.3  

Here's the error report and debug info: [vagrant 1.7.3 nfs synced folders error ](https://gist.github.com/christopher-hopper/ccef04464a5c1f13d44f).

**Comment by christopher-hopper** [comment]
@akostadinov, an update: 

This morning I installed the newer [vagrant 1.7.4](http://www.vagrantup.com/downloads). Before trying it out, I ensured my `/etc/exports` file was clean of old, orphaned vagrant entries. It seems to now be working as expected. 

I've tried the following:

- `vagrant up`
- `vagrant reload`
- `vagrant halt`

All add and clean-up their NFS Exports entries as expected using the `/etc/exports` file. 

Can we close this now?
